test/cli/errors-skip-cache/test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euo pipefail
+
+dir=$(mktemp -d)
+cleanup() {
+    rm -r "$dir"
+}
+trap cleanup EXIT
+
+mkdir "$dir/cache"
+
+run_sorbet() {
+  # Ideally this would include counters output so that we could verify the cache
+  # miss, but there are a lot of unstable metrics in the counters output, and
+  # filtering it means that we'll lose the errors that duplicate across sorbet
+  # invocations.
+  #
+  # TODO(trevor): update this to include counter output if we fix `--counter`
```

Comment:
I don't think that this actually matters so much—if we were to make a change to the cache such that we actually *were* able to have a cache hit but still have the errors show up on the warm cache, we wouldn't actually care to enforce that there _must_ be a cache miss on this example.

That is, I think that the structure of this test is sufficiently fine, absent the `--counter` flag not working.

Reply:
I've removed the comment.